easybuild/easyconfigs/k/kallisto/kallisto-0.45.0-foss-2018b.eb

```
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'CMakeMake'
+
+name = 'kallisto'
+version = '0.45.0'
+
+homepage = 'http://pachterlab.github.io/kallisto/'
+description = """kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
+ of target sequences using high-throughput sequencing reads."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/pachterlab/kallisto/archive/']
```

@wpoely86 You could use the `GITHUB_SOURCE` template here:

```suggestion
github_account = 'pachterlab'
source_urls = [GITHUB_SOURCE]
```

and maybe the `%(name)s` template instead of `kallisto` in some places.